content/docs/developer/anteaterapi/migrating.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Migrating from PeterPortal API
+---
+
+import { Steps, Step } from "fumadocs-ui/components/steps";
+
+PeterPortal API will be sunset **on Friday, January 31st, 2025, at 11:59 pm Pacific Standard Time**. After this time, requests made to either version of PeterPortal API will no longer be served.
+
+Anteater API is the successor to PeterPortal API. The sole upgrade path from either version of PeterPortal API is to Anteater API.
```

Comment:
Add something along the lines of, "In the time since we've built PPAPI we've learned a lot about the data we work with and it's use cases. Anteater API will allow us to more efficiently make this data accessible, and represents our work to continuously improving long term reliably."